Mathlib/FieldTheory/Galois/IsGaloisGroup.lean

```diff
@@ -151,6 +151,36 @@ instance of_isGalois [FiniteDimensional K L] [IsGalois K L] : IsGaloisGroup Gal(
   commutes := inferInstance
   isInvariant := ⟨fun x ↦ (IsGalois.mem_bot_iff_fixed x).mpr⟩
 
+variable {G}
+
+/--
+For `G` be a Galois group for `L/K` and `S ⊆ G` be a subgroup, the `IntermediateField` of the
+points in `L` fixed by elements of `S`.
+-/
+def fixedField [IsGaloisGroup G K L] (S : Subgroup G) : IntermediateField K L :=
+  FixedPoints.intermediateField S
```

Comment:
Maybe the solution here is to simply make the two fields explicit in `FixedPoints.intermediateField`?